README.md

```diff
@@ -148,6 +148,7 @@ Additional lists you might find useful:
 ## Email
 *Plugins for sending and parsing email.*
 
+- [Elastic Email plugin](https://github.com/pnglabz/cakephp-elastic-email) - Email transport plugin for sending email via Elastic Email API.
```

Comment:
Github URL seems to have changed now.

Reply:
Yes, the organization name is changed. Let me update.